src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    static final int COLOR_NUMBER = 3;
```

Comment:
You can use Color.values().length instead

Reply:
> You can use Color.values().length instead

Ok , thanks , I will fix it .